src/SessionManager/ProgramSettings.cc

```diff
@@ -76,7 +76,8 @@ ProgramSettings::ProgramSettings(int argc, char** argv) {
  3   warning
  4   info
  5   debug
- 6   trace)");
+ 6   trace
+ 7   trace + ICD)");
```

Comment:
Similarly, I think that we should rename this to something like `protocol` or `trace (protocol)`.

It's already implied that each level includes the levels above it.

(ICD is our internal term for a document.)

Reply:
I agree, protocol makes more sense than ICD